v5/Rules/StandardText.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace v5\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class StandardText implements Rule
+{
+    protected $patterns = [
+        // Regex that only allows letters, numbers, punctuation, and space.
+        '/^[\pL\pN\pP ]++$/uD',
+        // Regex for bengali characters
+        '/^[\p{Bengali} ]{0,100}$/u',
```

Comment:
What would happen if someone mixes character sets in their input? I.e. they write something in Bengali, but then in the middle of the sentence, they write i.e. "Twitter", because they are referring to it as a source.

Would that fail validation?

Reply:
Oops ... I tested this edge case, and it fails validation.

Reply:
@tuxpiper so I updated the regex to handle a wide case of characters

https://github.com/ushahidi/platform/pull/4514/files#diff-4d687cdb0cd21587e307890aa061024d77106c84dc4f4174933f90871434314dR12-R13

This works when someone mixes character sets in their titile.